core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/bigquery/BigqueryDatabaseService.java

```diff
@@ -0,0 +1,166 @@
+package io.metadew.iesi.connection.database.bigquery;
+
+import io.metadew.iesi.connection.database.DatabaseHandler;
+import io.metadew.iesi.connection.database.ISchemaDatabaseService;
+import io.metadew.iesi.connection.database.SchemaDatabaseService;
+import io.metadew.iesi.connection.database.connection.DatabaseConnectionHandler;
+import io.metadew.iesi.metadata.definition.MetadataField;
+import io.metadew.iesi.metadata.definition.connection.Connection;
+
+public class BigqueryDatabaseService extends SchemaDatabaseService<BigqueryDatabase> implements ISchemaDatabaseService<BigqueryDatabase> {
```

Comment:
Does Bigquery use the limit X offset Y format for paginated retrieval of rows? If not, you should override 'public String refactorLimitAndOffset(String query)'. See io.metadew.iesi.connection.database.oracle.OracleDatabaseConnectionService for example

Reply:
Bigquery uses limit x offset y syntax -- see https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax for more information